apps/librelingo_yaml_loader/librelingo_yaml_loader/yaml_loader.py

```diff
@@ -332,14 +334,49 @@ def _convert_license(raw_license):
     )
 
 
+def _convert_text_to_speech_settings(raw_audio_settings):
+    """
+    Creates an TextToSpeechSettings() object based on the data structure in the YAML
+    file
+    """
+    if "TTS" not in raw_audio_settings:
+        return [TextToSpeechSettings()]
+
+    return [
+        TextToSpeechSettings(tts["Provider"], tts["Voice"], tts["Engine"])
+            for tts in raw_audio_settings["TTS"]
+    ]
+
+
+def _convert_audio_settings(raw_settings):
+    """
+    Creates an AudioSettings() object based on the data structure in the YAML
+    file
+    """
+    if "Audio" not in raw_settings:
+        return AudioSettings()
+
+    raw_audio_settings = raw_settings["Audio"]
+
+    if raw_audio_settings["Enabled"]:
+        text_to_speech_settings = _convert_text_to_speech_settings(raw_audio_settings)
```

Comment:
```suggestion
        text_to_speech_configs = _convert_text_to_speech_settings(raw_audio_settings)
```

I feel like it's a better naming, no? Maybe throughout the codebase. If I understand correctly, there can be multiple configs simultaneously

Reply:
Is the issue here that "settings" is used for both the singular and the plural? "Configs" doesn't seem any clearer to me.

If I've understood you correctly, maybe `text_to_speech_settings_list` would be good?

Reply:
> Is the issue here that "settings" is used for both the singular and the plural? 

yeah, I think this was my issue yes 😅 even though maybe I haven't realized it consciously 

> If I've understood you correctly, maybe text_to_speech_settings_list would be good?

sounds perfect to me!